lib/MCT.pm

```diff
@@ -28,59 +28,29 @@ sub startup {
 
   $app->_setup_database;
   $app->_setup_secrets;
-  $app->_ensure_conference;
   $app->_routes;
-}
-
-sub _auto_routes {
```

Comment:
Moved this to ```MCT::Controller::Conference::page()```.
Also moved the templates from "templates/auto" to "templates/2015".